application-task-test/application-task-test-docker/src/test/it/com/xwiki/task/test/ui/AllITs.java

```diff
@@ -30,11 +33,20 @@
  * @since 2.6
  */
 @UITest
+@TestClassOrder(ClassOrderer.OrderAnnotation.class)
```

Comment:
Why do you need this?

Reply:
Check https://github.com/xwikisas/application-task/pull/178#discussion_r1967201771